src/app.module.ts

```diff
@@ -1,10 +1,11 @@
 import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { MensajesController } from './mensajes/mensajes.controller';
```

Comment:
Try not to mix `english` and `spanish` when naming variables or functions

please update all to English